tests/foreman/cli/test_leapp_client.py

```diff
@@ -295,86 +295,3 @@ def test_leapp_upgrade_rhel(
     custom_leapp_host.clean_cached_properties()
     new_ver = str(custom_leapp_host.os_version)
     assert new_ver == upgrade_path['target_version']
-
-
-@pytest.mark.parametrize(
-    'upgrade_path',
-    [
-        {'source_version': RHEL8_VER, 'target_version': RHEL9_VER},
-    ],
-    ids=lambda upgrade_path: f'{upgrade_path["source_version"]}'
-    f'_to_{upgrade_path["target_version"]}',
-)
-def test_leapp_upgrade_rhel_non_admin(
-    module_target_sat,
-    module_sca_manifest_org,
-    default_location,
-    custom_leapp_host,
```

Comment:
we recently added this test, why removing this now for 6.14.z?

Reply:
It was removed by mistake.